spec/support/controller_helpers.rb

```diff
@@ -0,0 +1,16 @@
+module ControllerHelpers
+  def sign_in(user = create_arlington_user)
```

Comment:
Doesn't this come with devise? Or by including Devise helpers in a controller test? I could be wrong.